ProgressBar/themes/bootstrap/variables.less

```diff
@@ -1,2 +1,3 @@
-@transition-params: 0.3s linear 0s;
-@line-height: 1.6em;
\ No newline at end of file
+@height: 1.8em; // Controls the height of the widget (180% of the font-size)
```

Comment:
> Can this collapse with other widget defining a similar variable?

On a related note, shouldn't we systematically apply a naming pattern (with the "d-" prefix) also for CSS classes? I'm not sure whether we have other cases, the only one I have in mind is the `fill` class (https://github.com/ibm-js/deliteful/blob/0.6.0-alpha/LinearLayout/themes/LinearLayout_template.less#L12). I can create an issue for it.